Fix bug report confirmation dialog width and scroll

Parthiv-M opened the pull request:
- EDIT: Previously, the message returned a string enclosed in `` with fixed spacing/new lines. This has been changed to be concatenated with `+` s so that a ```<p />``` is generated instead of a `pre` which was causing the fixed width and ugly scroll issue
- Fixes #5559

thecodrr commented:
This makes no sense. \`\` doesn't produce `pre` tag. It's just used for string substitution. Changing it to "" doesn't make a difference. Have you actually tested this change?

thecodrr commented on apps/web/src/dialogs/issue-dialog.tsx:
```diff
@@ -150,11 +150,11 @@ function showIssueReportedDialog({ url }: { url: string }) {
-    Please note that we will respond to your bug report on the link above. **We recommended that you save the above link for later reference.**
-    
-    If your issue is critical (e.g. notes not syncing, crashes etc.), please [join our Discord community](https://discord.com/invite/zQBK97EE22) for one-to-one support.`
+    message: "You can track your bug report at [" + url + "](" + url + ")."
+
+      + "\n\n<br />Please note that we will respond to your bug report on the link above. **We recommended that you save the above link for later reference.**"
+
+      + "\n\n<br />If your issue is critical (e.g. notes not syncing, crashes etc.), please [join our Discord community](https://discord.com/invite/zQBK97EE22) for one-to-one support."
```
This is unnecessary. The pre tag is created due to the indentation. Just remove that and it'll work.

To add empty line breaks in Markdown, use `\\` instead of `<br>`.

Parthiv-M commented:
> This makes no sense. `` doesn't produce `pre` tag. It's just used for string substitution. Changing it to "" doesn't make a difference. Have you actually tested this change?

Yep, it worked in my tests (locally). But I'll make the indentation change nevertheless.

Parthiv-M commented:
Hi, I realised that the way I interpreted the change was wrong, rather than the usage of backticks itself. The code that fixed it was the concatenation of strings rather than removing interpolation altogether. 

I have reverted back to using backticks but concatenated them instead of giving fixed spacing. 

My apologies for the confusion. I have attached a screenshot of the result and updated the PR description too.

![bug_report](https://github.com/user-attachments/assets/3d311060-cd0c-418a-9017-d0c32d2915a0)

thecodrr commented on apps/web/src/dialogs/issue-dialog.tsx:
```diff
@@ -150,11 +150,9 @@ function showIssueReportedDialog({ url }: { url: string }) {
-    Please note that we will respond to your bug report on the link above. **We recommended that you save the above link for later reference.**
-    
-    If your issue is critical (e.g. notes not syncing, crashes etc.), please [join our Discord community](https://discord.com/invite/zQBK97EE22) for one-to-one support.`
+    message: `You can track your bug report at [${url}](${url}).`
+      + `\n\nPlease note that we will respond to your bug report on the link above. **We recommended that you save the above link for later reference.**`
+      + `\n\nIf your issue is critical (e.g. notes not syncing, crashes etc.), please [join our Discord community](https://discord.com/invite/zQBK97EE22) for one-to-one support.`
```
```suggestion
    message: `You can track your bug report at [${url}](${url}).

\
Please note that we will respond to your bug report on the link above. **We recommended that you save the above link for later reference.**

\
If your issue is critical (e.g. notes not syncing, crashes etc.), please [join our Discord community](https://discord.com/invite/zQBK97EE22) for one-to-one support.`
```

thecodrr commented:
Thanks!
